_data/menus.yml

```diff
@@ -7,6 +7,9 @@ items:
         extra_path: "/contribute"
         children:
           - title: Contribute
+            children:
+              - title: Docs Guide
+                extra_path: "/docs-guide"
```

Comment:
as an fyi, you do not need the `extra_path` in this case, as it automatically creates the path "docs-guide". `extra_path` is just needed for creating the "previous path" for child elements (sounds confusing, happy to explain further if it's unclear)

anyway, I'll quickly remove this on the `master` branch